03-task-manager/starter/quizAnswers.txt

```diff
@@ -0,0 +1,12 @@
+(1) Quickly explain how middleware is used in an Express application.
+-- Middleware is/are functions automatically initialized when following a designated path.
```

Comment:
Also worth noting that middleware in express runs in the context of a request, often between request receival and response generation, and is helpful for centralizing common operation